scripts/nb-tester/test-notebook.py

```diff
@@ -50,7 +51,7 @@ def filter_paths(paths: list[Path], args: argparse.Namespace) -> Iterator[Path]:
     """
     Filter out any paths we don't want to run, printing messages.
     """
-    submit_jobs = args.submit_jobs or args.only_submit_jobs
+    submit_jobs = args.submit_jobs or args.only_unmockable
```

Comment:
It's confusing to call the variable `submit_jobs` since it looks like `args.submit_jobs`. What about `use_hardware`?

Reply:
What do you think of 90a348b238a54e4a4ac18e42ccd7b808cf1344a2 as a slightly different solution? It sets `args.submit_jobs` to `True` if `only_unmockable` is set, which lets us remove a few of these lines.

Reply:
I like that. Maybe worth editing the argparse docstring to say

> Setting this option implicitly sets `--submit-jobs`.